Xamarin.Forms.Platform.WPF/FormsRadioButton.cs

```diff
@@ -0,0 +1,72 @@
+﻿using WContentPresenter = System.Windows.Controls.ContentPresenter;
+using System.Windows;
+using System.Windows.Media;
+using Xamarin.Forms.Platform.WPF.Controls;
+
+namespace Xamarin.Forms.Platform.WPF
+{
+	public class FormsRadioButton : System.Windows.Controls.RadioButton
+	{
+		public static readonly DependencyProperty BorderRadiusProperty = DependencyProperty.Register(nameof(BorderRadius), typeof(int), typeof(FormsButton),
+	new PropertyMetadata(default(int), OnBorderRadiusChanged));
+
+		public static readonly DependencyProperty BackgroundColorProperty = DependencyProperty.Register(nameof(BackgroundColor), typeof(Brush), typeof(FormsButton),
+			new PropertyMetadata(default(Brush), OnBackgroundColorChanged));
+
+		
+		public Brush BackgroundColor
+		{
+			get
+			{
+				return (Brush)GetValue(BackgroundColorProperty);
+			}
+			set
+			{
+				SetValue(BackgroundColorProperty, value);
+			}
+		}
+
+		public int BorderRadius
+		{
+			get
+			{
+				return (int)GetValue(BorderRadiusProperty);
+			}
+			set
+			{
+				SetValue(BorderRadiusProperty, value);
+			}
+		}
+
+		public override void OnApplyTemplate()
+		{
+			base.OnApplyTemplate();
+
+			UpdateBackgroundColor();
+			UpdateBorderRadius();
+		}
+
+		static void OnBackgroundColorChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+		{
+			((FormsRadioButton)d).UpdateBackgroundColor();
+		}
+
+		static void OnBorderRadiusChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+		{
+			((FormsRadioButton)d).UpdateBorderRadius();
+		}
+
+		void UpdateBackgroundColor()
+		{
+			if (BackgroundColor == null)
+				BackgroundColor = Background;
+
+			Background = BackgroundColor;
+		}
+
+		void UpdateBorderRadius()
```

Comment:
This empty method does not seem necessary, we can eliminate it.

Reply:
Done